iroha_macro/iroha_derive/src/lib.rs

```diff
@@ -9,6 +9,8 @@ use syn::{
     PatIdent, PatReference, PatStruct, PatTuple, PatTupleStruct, PatType, Signature,
 };
 
+const SKIP_FROM_ATTR: &str = "skip_from";
```

Comment:
Please add some docs for this attribute. Possibly at the place of implementation of `FromVariant`